api/client.go

```diff
@@ -7,6 +7,8 @@ import (
 	"time"
 )
 
+const baseUrl = "http://www.ctabustracker.com/bustime/api/v2"
```

Comment:
Same as below with the args / env vars. Additionally, this should ideally be injected to the api client